src/Microsoft.OData.Core/UrlValidation/Rules/DeprecationRules.cs

```diff
@@ -271,7 +271,7 @@ private static IEnumerable<IEdmVocabularyAnnotation> GetRevisionAnnotations(IEdm
         private static bool isRevisionsAnnotation(IEdmVocabularyAnnotation annotation)
         {
             return string.Equals(annotation.Term.FullName(), CoreVocabularyConstants.Revisions, StringComparison.OrdinalIgnoreCase) ||
-                            string.Equals(annotation.Term.FullName(), DefaultCoreAlias + "." + RevisionTerm, StringComparison.OrdinalIgnoreCase);
+                            string.Equals(annotation.Term.FullName(), string.Concat(DefaultCoreAlias, ".", RevisionTerm), StringComparison.OrdinalIgnoreCase);

```

Comment:
> string.Concat(DefaultCoreAlias, ".", RevisionTerm) [](start = 69, length = 51)

again, since all values are constants, consider defining a private const string QualifiedRevisionsTerm =  string.Concat(DefaultCoreAlias, ".", RevisionTerm)